package.json

```diff
@@ -29,18 +53,31 @@
     "eslint-plugin-react": "^7.3.0",
     "html-webpack-plugin": "^2.30.1",
     "install": "^0.10.1",
+    "jest": "^21.1.0",
     "nodemon": "^1.12.1",
     "npm": "^5.4.2",
     "react": "^15.6.1",
     "react-dev-utils": "^4.0.1",
+    "react-dom": "^15.6.1",
+    "react-test-renderer": "^15.6.1",
     "webpack": "^3.6.0",
     "webpack-bundle-analyzer": "^2.9.0",
     "webpack-dashboard": "^1.0.0-6",
     "webpack-dev-server": "^2.8.2",
     "webpack-merge": "^4.1.0"
   },
   "dependencies": {
-    "britecharts": "^2.0.1",
-    "clean-webpack-plugin": "^0.1.16"
+    "britecharts": "^2.0.1"
+  },
+  "jest": {
+    "moduleFileExtensions": [
```

Comment:
i'm not sure you really need this right now right?

Reply:
I am not sure!
In https://facebook.github.io/jest/docs/en/webpack.html#configuring-jest-to-find-our-files it says that this is equivalent to 'extensions' in webpack.
It seems to work without it!